<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoWindow</class>
 <widget class="QWidget" name="InfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="humLabel">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>302</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lightLabel">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>76</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tempLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>185</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="plantGIFLabel">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="gotoMainPageButton">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>210</y>
     <width>71</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border :none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>367</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Images/GreenHouse_UI_Design/Background/info-page.jpg</pixmap>
   </property>
  </widget>
  <zorder>backgroundLabel</zorder>
  <zorder>humLabel</zorder>
  <zorder>lightLabel</zorder>
  <zorder>tempLabel</zorder>
  <zorder>plantGIFLabel</zorder>
  <zorder>gotoMainPageButton</zorder>
  <zorder>dateLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
